src/wallet/wallet2.cpp

```diff
@@ -3076,6 +3113,32 @@ crypto::hash8 wallet2::get_short_payment_id(const pending_tx &ptx) const
 void wallet2::commit_tx(pending_tx& ptx)
 {
   using namespace cryptonote;
+  
+  if(m_light_wallet) {
+    
+    cryptonote::COMMAND_RPC_LIGHT_WALLET_SUBMIT_RAW_TX::request oreq;
+    cryptonote::COMMAND_RPC_LIGHT_WALLET_SUBMIT_RAW_TX::response ores;
+    
+    oreq.address = get_account().get_public_address_str(m_testnet);
+    oreq.view_key = string_tools::pod_to_hex(get_account().get_keys().m_view_secret_key);
+    oreq.tx = epee::string_tools::buff_to_hex_nodelimer(tx_to_blob(ptx.tx));
+    m_daemon_rpc_mutex.lock();
+    bool r = epee::net_utils::invoke_http_json("/submit_raw_tx", oreq, ores, m_http_client, std::chrono::seconds(5), "POST");
+    m_daemon_rpc_mutex.unlock();
+    if (!r) {
+      MERROR("Lightwallet: Failed to submit raw tx");
+    }
```

Comment:
Using THROW_WALLET_EXCEPTION_IF instead since that seems to be used with invoke_http_json in other places. throw_on_rpc_response_error expects a string as argument.